src/main/java/org/terracotta/statistics/AbstractOperationStatistic.java

```diff
@@ -63,9 +65,22 @@ public long sum() {
   public void begin() {
     if (!derivedStatistics.isEmpty()) {
       long time = Time.time();
+      operationStartTime.set(time);
```

Comment:
Do we want to necessarily calculate latency for any operation? Because it seems to be the case here (unless there are no derived statistics)

Reply:
no, if you read the PR description, this is not calculated for each operation.

Reply:
Yes. That's what I meant. But by reader lower, it seems it was the current behavior anyway.